templates/item/preview.macros.njk

```diff
@@ -293,8 +293,12 @@
   {% if width or text %}
     <td aria-hidden data-herman-cell="viz" {% if colspan %}colspan="{{ colspan }}"{% endif %}>
       <div data-herman-size="overflow">
-      {% if width %}
-        <div data-herman-viz="bar" style="width: {{ width }};">
+        {% if width %}
+          {% if value in ['thin', 'thick', 'medium'] %}
+            <div data-herman-viz="bar" data-herman-border style="border-width: {{ width }};">
```

Comment:
Might be cleaner with something like `data-herman-viz="bar border"`? But this works great.

Reply:
I had it like that at first but then changed it to be a separate attribute. I don't feel strongly either way.